content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/reordering_rules.rst

```diff
@@ -79,41 +83,179 @@ Reordering Rules`, then click :guilabel:`Create`, and fill out the new line as f
 
 .. tip::
    Reordering rules can also be created from each product form. To do so, navigate to
-   :menuselection:`Inventory --> Products --> Products`, then select a product. Click on
-   :menuselection:`Reordering Rules --> Create`, then fill out the new line as detailed above.
+   :menuselection:`Inventory app --> Products --> Products`, then select a product. Click the
+   :guilabel:`Reordering Rules` smart button, then :guilabel:`New` to fill out the new line as
+   detailed above.
+
+For advanced usage of reordering rules, learn about the following reordering rule fields:
+
+#. :ref:`route <inventory/product_management/route>`
+#. :ref:`trigger <inventory/product_management/trigger>`
+#. :ref:`visibility days <inventory/product_management/visibility-days>`
+
+.. _inventory/product_management/trigger:
+
+Trigger
+=======
+
+When stock falls below the reordering rule's minimum, set the reordering rule's *trigger* to
+*automatic* to automatically create purchase or manufacturing orders to replenish stock.
+
+On the other hand, setting the reordering rule's trigger to *manual* displays the product and
+forecasted stock on the *replenishment dashboard*, where the procurement manager an review the stock
+levels, lead times, and forecasted dates of arrival.
+
+.. seealso::
+   :doc:`strategies`
+
+.. tip::
+   The replenishment dashboard is accessed by going to :menuselection:`Inventory app --> Operations
+   --> Replenishment`.
+
+To enable the :guilabel:`Trigger` field, go to :menuselection:`Inventory app --> Configuration -->
+Reordering Rules`. Then, click the :guilabel:`(slider)` icon and enable the :guilabel:`Trigger`
+option from the additional options drop-down menu.
 
-Once a reordering rule has been created for a product, if its forecasted quantity is below the
-rule's minimum quantity when the scheduler runs, a replenishment order for the product is
-automatically generated. By default, the scheduler runs once each day.
+.. image:: reordering_rules/enable-trigger.png
+   :align: center
+   :alt: Enable the Trigger field by toggling it in the additional option menu
```

Comment:
```suggestion
   :alt: Enable the Trigger field by toggling it in the additional options menu
```
Nitpicky I know haha

Reply:
Thanks for catching this! I'd call this discerning, not nitpicky (: